modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -413,39 +514,49 @@ export default class ortcRTCPeerConnection extends yaeti.EventTarget {
 
     /**
      * Creates a RTCDataChannel.
-     * TBD
      */
     createDataChannel() {
         logger.debug('createDataChannel()');
+
+        // TODO: WebSocket hack not yet done. For now let's throw.
+
+        throw new Error('createDataChannel() not supported in Edge');
     }
 
     /**
      * Gets a sequence of local MediaStreams.
+     * @return {sequence<MediaStream>}
      */
     getLocalStreams() {
-        return Array.from(this._localStreams);
+        return Array.from(this._localTrackInfos.values())
+            .map(info => info.stream)
+            .filter((elem, pos, arr) => arr.indexOf(elem) === pos);
     }
 
     /**
      * Gets a sequence of remote MediaStreams.
+     * @return {sequence<MediaStream>}
      */
     getRemoteStreams() {
-        return Array.from(this._remoteStreams);
+        return Array.from(this._remoteStreams.values());
     }
 
     /**
-     * TBD
+     * Get RTP statistics.
      */
     getStats() {
-        // TBD
+        logger.debug('getStats()');
+
+        // TODO: TBD
+
+        throw new Error('getStats() not yet implemented');
```

Comment:
It depends on the app usage. Old WebRTC API throws when something is wrong, or sometimes invokes the given errback. I'll complete it with both a callback/errback and promise capable signature, as all the other methods.